webcomponents/search-webcomponent/src/app/shared/form/form-field-control.service.spec.ts

```diff
@@ -36,5 +41,27 @@ describe('FormFieldControlService', () => {
     const formFields = [formField1, formField2, formField3, formField4];
     const formGroup = service.toFormGroup(formFields);
     expect(Object.keys(formGroup.value).length).toEqual(2);
+    expect(formGroup).toBeInstanceOf(FormGroup);
+  });
+
+  it('should create validation messages', () => {
```

Comment:
Maybe give another detail in the title? Such as, when/under which circumstance the validation message is supposed to appear